cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -853,6 +854,124 @@
     </description>
   </property>
 
+  <property>
+    <name>kafka.bind.address</name>
+    <value>0.0.0.0</value>
+    <description>
+      DEPRECATED: CDAP Kafka service bind address
```

Comment:
Rather than highlighting with all-caps (which looks garish), please follow as the others in this file, and add it at the end, such as:
```
CDAP Kafka service bind address (deprecated; replaced with kafka.server.bind.address)